asmcli/lib/iam.sh

```diff
@@ -0,0 +1,148 @@
+required_iam_roles_mcp_sa() {
```

Comment:
About the file name: there's a lot of stuff in here that doesn't have anything to do with IAM. Maybe grouping all of the prerequisite requirements and associated functions into one file would be a better first step? Just IAM seems too narrow.

Reply:
Put it to `installation_dependencies.sh`